policyengine_us/parameters/gov/states/ne/tax/income/agi/subtractions/military_retirement/fraction.yaml

```diff
@@ -0,0 +1,13 @@
+description: Fraction of military retirement benefits allowed as NE AGI subtraction.
+values:
+  2022-01-01: 1
```

Comment:
could we add 2021 value as well?

Reply:
<img width="690" alt="Screenshot 2024-09-12 at 11 08 09" src="https://github.com/user-attachments/assets/c01e7530-44ca-47ba-80b6-f8c9a4fba28c">
On the 2021 tax form, the percentage is 40%. However, in the legal code, from 2015 to 2021, the tax filer could choose between receiving 40% for 7 years or 15% for all years after age 67. I'm not sure how to represent this. 
<img width="816" alt="Screenshot 2024-09-12 at 11 09 38" src="https://github.com/user-attachments/assets/1b320628-f8a2-4020-8674-5116b761d95a">

Reply:
Lets represent this as the 40% described in the tax form and add a note in the variable